docs/DMS/PRIMARY-DATA-STORAGE/RELATIONAL-QUERIES.md

```diff
@@ -1,16 +1,20 @@
-# DMS Feature: Queries Using the Relational Database
+# DMS Feature: Relational Database Queries
 
 > [!TIP]
 > See [DMS Feature: Primary Data Storage](./README.md) for detailed background on the database design.
 
-While the preferred method of query handling is via [search engine](../SEARCH-DATABASE.md), some
-deployments will not be able to handle the additional operational complexity. In these cases DMS can
-be configured to handle queries in the main datastore at a cost of performance.
+While the preferred method for query handling is via a [search engine](../SEARCH-DATABASE.md), some deployments will not be able to handle the additional operational complexity. In these cases, DMS can be configured to handle queries directly in the main relational datastore, though this comes at a cost of query performance.
```

Comment:
> "... comes at a cost of query performance."

Have we proven that statement? I know that's an old statement, but since we're in here, I wonder if we should remove it since it is really just a hypothesis at this point.

Reply:
I can be more explicit, but the idea is that you're going to have some balance of indexing on all the possible queryable fields. More indexes = slower insert performance. Less indexes = Poor query performance when non-indexed fields used.